packages/client/src/components/textarea/textarea.tsx

```diff
@@ -0,0 +1,72 @@
+import classNames from 'classnames';
+import { ChangeEvent, useRef, useState } from 'react';
+import { IValidationInputProps } from '../../types/validation';
+import style from './textarea.module.scss';
+
+interface Props extends IValidationInputProps {
+  value: string;
+  placeholder?: string;
+  label?: string;
+  solo?: boolean;
+  resize?: 'none' | 'both' | 'horizontal' | 'vertical' | 'inherit';
+  onChange?: (event?: ChangeEvent<HTMLTextAreaElement>) => void;
+  onBlur?: (event?: ChangeEvent<HTMLTextAreaElement>) => void;
+}
+
+function Textarea(props: Props) {
+  const { value, onChange, solo, label, onBlur, resize, errorText } = props;
+
+  const [labelFocus, setLabelFocus] = useState(!!value);
+
+  const textareaRef = useRef<HTMLTextAreaElement>(null);
+
+  const focusTextareaContainer = () => {
+    if (textareaRef.current) {
+      setLabelFocus(true);
+      textareaRef.current.focus();
+    }
+  };
+
+  const onBlurTextarea = (event: ChangeEvent<HTMLTextAreaElement>) => {
+    if (!value) {
+      setLabelFocus(false);
+    }
+    if (onBlur) {
+      onBlur(event);
+    }
```

Comment:
[Предложение]
Можно сократить запись до 
```suggestion
    !value && setLabelFocus(false);
    onBlur && onBlur(event);
```

Reply:
с этим не согласен?